libs/next_gen_ui_a2a/agent_card.py

```diff
@@ -0,0 +1,24 @@
+from a2a.types import AgentCapabilities, AgentCard, AgentSkill
+
+skill = AgentSkill(
```

Comment:
We should probably provide some way how to define `skill` and `card` content in the agent configuration, as it is important for final AI app tuning. Maybe create another jira for it and implement later.

Reply:
Good point https://issues.redhat.com/browse/NGUI-497